frontend/src/features/admin-form/create/end-page/EndPageDrawer.tsx

```diff
@@ -24,20 +24,22 @@ import Textarea from '~components/Textarea'
 import { useMutateFormPage } from '~features/admin-form/common/mutations'
 import { useAdminForm } from '~features/admin-form/common/queries'
 
+import { DrawerContentContainer } from '../builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/common/DrawerContentContainer'
```

Comment:
nit: super long import statement :((((, should we keep creating `index.ts`s to import from `../builder-and-design/BuilderAndDesignDrawer` instead?

Reply:
moved DrawerContentContainer into common instead :)